.travis.yml

```diff
@@ -1,15 +1,5 @@
 language: python
 
-services:
```

Comment:
I just looked at the travis file (not the repo name), which contains a mongo server. Since this is portable_mds, there is no need for travis to build a mongo server.